protocol/dubbo/dubbo_invoker.go

```diff
@@ -196,8 +196,12 @@ func (di *DubboInvoker) Destroy() {
 		di.BaseInvoker.Destroy()
 		client := di.getClient()
 		if client != nil {
+			activeNumber := client.DecreaseActiveNumber()
 			di.setClient(nil)
-			client.Close()
+			if activeNumber == 0 {
```

Comment:
- If destroy method is responsible for client close, then we should close client whatever active number equals or not equals to zero.
- If destroy method is not responsible for client close, then we should not close client in destroy method.

So 

- Should we close client in destroy method? 
- If we should, should we close client after active number check?

Reply:
Good question,  for the first question:
1.Should we close client in destroy method?
Yes, now the design is one node(VM)<=>one ExchangeClient. So we need close the tcp conn when all the invokers destroyed.
2.If we should, should we close client after active number check?
As the answer in the first question, we need close the conn until all the invokers(in one Node(VM)) destroyed.

Reply:
So now codes seems to implement the (2), not (1)? I'm not very sure.

(1) close the conn until all the invokers (in one Node(VM)) destroyed
(2) close the conn if activenumber is zero

Reply:
Im my opinion,(1) also same with (2), and we need check the activeNumber when try to close the global(one VM) conn

Reply:
So we only need to check that active number equals to zero (nonblocking), instead of waiting until active number becomes to zero (blocking)?

Reply:
now i think here is nonblocking, do you think it is blocking here？

Reply:
I am not sure. If tests cover this case, I think there is no problem to some extend.